test-harness/build.gradle

```diff
@@ -37,3 +37,8 @@ uploadArchives {
     }
   }
 }
+
+// Module name should be reverse-DNS (com.github.spotbugs) just like package name
+// http://blog.joda.org/2017/04/java-se-9-jpms-module-naming.html
+ext.moduleName = 'com.github.spotbugs.test'
```

Comment:
Schouldn't this be .test.harness?

Reply:
I think .test is OK, we use `edu.umd.cs.findbugs.test` as package name.